source/core/lethe_grid_tools.cc

```diff
@@ -594,7 +594,7 @@ LetheGridTools::project_to_d_linear_object(
   //                   \partial_m phi_j
   // we start at xi=(0.5, 0.5).
   Point<structdim> xi;
-  for (unsigned int d = 0; d < structdim; ++d)
+  for (unsigned int d = 0; std::cmp_less(d, structdim); ++d)
```

Comment:
Why not make d an int instead here?
I don't understand why we need the cmp_less (which is tailored made for two different type here....